README.rst

```diff
@@ -129,8 +129,6 @@ Development is supported by the Research Council of Norway as a part of
 
 .. |Build Status| image:: https://travis-ci.org/nansencenter/nansat.svg?branch=master
    :target: https://travis-ci.org/nansencenter/nansat
-.. |AppVeyor Status| image:: https://ci.appveyor.com/api/projects/status/la50x7l2yy4d9ljr/branch/master?svg=true
```

Comment:
Was this used for testing under windows, which we agreed to stop supporting?

Reply:
Yes